repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/GatherTestInformation4Bears.java

```diff
@@ -0,0 +1,29 @@
+package fr.inria.spirals.repairnator.process.step;
+
+import fr.inria.spirals.repairnator.process.ProjectState;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector4Bears;
+
+public class GatherTestInformation4Bears extends GatherTestInformation {
+
+	public GatherTestInformation4Bears(ProjectInspector inspector) {
+		super(inspector);
+	}
+
+	@Override
+    protected void businessExecute() {
+		super.businessExecute();
+		
+    	if (!((ProjectInspector4Bears)this.inspector).isAboutAPreviousBuild()) {
+    		this.shouldStop = !this.shouldStop;
```

Comment:
you should not use that kind of casting: if I ever want to use a step you make I'll have a runtime error. 
If you really need those methods, add them in the ProjectInspector.